torch/csrc/jit/import_source.cpp

```diff
@@ -184,23 +189,56 @@ struct SourceImporter {
 
       std::vector<Def> definitions;
       std::vector<ResolverPtr> resolvers;
-      auto class_def = ClassDef(p_.parseClass());
-      for (const auto& method_def : class_def.defs()) {
-        definitions.emplace_back(method_def);
-        resolvers.emplace_back(resolver_);
-      }
+      auto parsed_treeref = p_.parseClass();
```

Comment:
parseClassLike? make it more clear it might return a named tuple or an interface.